src/App.js

```diff
@@ -1,19 +1,130 @@
 import React from 'react';
+import { TodoList } from './components/TodoList/TodoList';
+
 import './App.css';
 
 import users from './api/users';
+import todos from './api/todos';
+
+const preparedList = todos.map(todo => ({
+  ...todo,
+  user: users.find(user => user.id === todo.userId),
+}));
+
+class App extends React.Component {
+  state = {
+    preparedTodos: [...preparedList],
+    usersSelect: '',
+    title: '',
+    titleError: false,
+    selectError: false,
+  }
+
+  dataChange = (event) => {
```

Comment:
```suggestion
  handleChange = (event) => {
```
read this guide ;)
https://github.com/kettanaito/naming-cheatsheet